react/features/invite/functions.js

```diff
@@ -612,6 +612,7 @@ export function getDialInfoPageURLForURIString(
     }
     const { protocol, host, contextRoot, room } = parseURIString(uri);
 
+    // TODO: do we care for releases here
```

Comment:
Why not add it for consistency? We might need it one day?

Reply:
Done.